query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/converters/PinotFunctionConverter.java

```diff
@@ -80,9 +80,8 @@ private String functionToStringForAvgRate(
       java.util.function.Function<Expression, String> argumentConverter,
       ExecutionContext executionContext) {
 
-    String columnName = argumentConverter.apply(function.getArgumentsList().get(0));
-    String rateIntervalInIso =
-        function.getArgumentsList().get(1).getLiteral().getValue().getString();
```

Comment:
That is, change this to something equivalent to 
```java
  function.getArgumentsList().size() == 2 
    ? function.getArgumentsList().get(1).getLiteral().getValue().getString() : DEFAULT_AVG_RATE_SIZE;
```

Reply:
made the change

Reply:
resolved